src/software/util/make_enum_macro/BUILD

```diff
@@ -0,0 +1,15 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "make_enum_macro",
+    hdrs = ["make_enum_macro.h"],
+)
```

Comment:
any particular reason we added `_macro` as a suffix? That seems like it's a bit out of line with our current naming?

Reply:
I was just following the previous name of `printable_enum_macro`. I'm fine changing it. Do you think just `make_enum` is better? Or `custom_enum` or something else?

Reply:
> I was just following the previous name of printable_enum_macro. 

Ah, gotcha.

I think `make_enum` makes the most sense here because if someone is trying to figure out where something (like `MAKE_ENUM`) is defined, it makes sense the file would have the same name as the thing defined in it (or at least a similar name).

Reply:
changed to `make_enum`